packages/main/src/SegmentedButton.js

```diff
@@ -0,0 +1,158 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import ResizeHandler from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+
+// Template
+import SegmentedButtonTemplate from "./generated/templates/SegmentedButtonTemplate.lit.js";
+
+// Styles
+import SegmentedButtonCss from "./generated/themes/SegmentedButton.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-segmentedbutton",
+	properties: /** @lends sap.ui.webcomponents.main.SegmentedButton.prototype */  {},
+	slots: /** @lends sap.ui.webcomponents.main.SegmentedButton.prototype */ {
+
+		/**
+		 * Defines the buttons of <code>ui5-segmentedbutton</code>.
+		 * <br><br>
+		 * <b>Note:</b> Multiple buttons are allowed.
+		 * <br><br>
+		 * <b>Note:</b> Use the <code>ui5-togglebutton</code> for the intended design.
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			propertyName: "buttons",
+			type: HTMLElement,
+			individualSlots: true,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.SegmentedButton.prototype */ {
+
+		/**
+		 * Fired when the pressed button changes.
+		 *
+		 * @event
+		 * @param {HTMLElement} pressedButton the pressed button.
+		 * @public
+		 */
+		pressChange: {
```

Comment:
Was this name agreed upon? Press change sounds ok for the toggle button, but for segmented button it's more like "selectionChange". How is it in OpenUI5?